lite/helpers.go

```diff
@@ -112,10 +112,11 @@ func genHeader(chainID string, height int64, txs types.Txs,
 	vals *types.ValidatorSet, appHash []byte) *types.Header {
 
 	return &types.Header{
-		ChainID: chainID,
-		Height:  height,
-		Time:    time.Now(),
-		NumTxs:  len(txs),
+		ChainID:  chainID,
+		Height:   height,
+		Time:     time.Now(),
+		NumTxs:   int64(len(txs)),
```

Comment:
should this be int64? int32 should be more than enough

Reply:
or it's to avoid int64(..) cast?

Reply:
yup, so it is the same type as TotalTxs. No need to have them different types, just confusion when adding